Assets/MRTK/SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -30,12 +30,6 @@ public class PokePointer : BaseControllerPointer, IMixedRealityNearPointer
         /// </summary>
         private const int maximumTouchableVolumeSize = 1000;
 
-        [SerializeField]
-        protected LineRenderer line;
-
-        [SerializeField]
-        protected GameObject visuals;
-
```

Comment:
Would the removal of these properties be considered a breaking change for people who have potentially extended the poke pointer and utilized these properties?  Or is this a very unlikely scenario?

Reply:
I think it should be unlikely, given that we've disabled the referenced gameobjects on our prefab in the first place.

Reply:
> given that we've disabled the referenced gameobjects on our prefab in the first place.

I think them being disabled by default in our prefab is a bit of a red herring as far as asserting that this feature isn't used. This PR is removing the code that will quickly enable / disable the visuals based on the active state of the pointer itself (as in, is it close to a touchable object): 

![image](https://user-images.githubusercontent.com/3580640/115615917-65221200-a2a4-11eb-9a2e-fb65d2b5ecf0.png)

My guess is they're disabled by default in the prefab because the most likely condition is that you aren't bringing your hand up immediately in the vicinity of a touchable object, but more generally just up in space. Enabling them in the prefab shouldn't affect the underlying behavior or visualization at runtime.

Reply:
In addition to that, the lineRenderer, which is the only actual visual component of that game object, is disabled by default on the prefab and never gets re-enabled in the code.

Plus, this work is needed to separate out the pointer "model" from the visuals code in the future anyways.

Reply:
I feel this feature could be used and still wonder if adding null checks would be preferable to removing the option.

Reply:
Maybe we could mark them obsolete instead of getting rid of them entirely?

Reply:
Part of the original reasons this bug was raised was because of the null checks though

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/253cef41b15b0fc2263fb2f4c08bf964f76b4910/Assets/MRTK/SDK/Features/UX/Scripts/Pointers/PokePointer.cs#L496-L497

Reply:
> the lineRenderer, which is the only actual visual component of that game object, is disabled by default on the prefab and never gets re-enabled in the code.

This is true. Feels more like an oversight to me than a reason to remove the ability to use the fields entirely though.

> Plus, this work is needed to separate out the pointer "model" from the visuals code in the future anyways

Is this a v2 plan or something for v3 where we'd be more able to take breaking changes?

Reply:
> Part of the original reasons this bug was raised was because of the null checks though

Those asserts could be removed, yeah. I more mean null checks before they're actually used, like

```c#
if (visuals != null)
{
    visuals.SetActive(IsActive);
}
```

Reply:
I think having vestigial code which requires users to instantiate gameobjects that aren't intended to be used is a less than desirable state, and I think we'll make things harder for use to maintain in the future by leaving in functionality that we didn't intend to use ourselves.